design-documents/graph-ql/coverage/Wishlist.graphqls

```diff
@@ -0,0 +1,77 @@
+type Query {
+    wishlist: WishlistOutput #already implemented
+}
+
+type Mutation {
+    createWishlist: ID @doc(description:"If not exists - creates new wishlist. If exists - returns wishlist ID for customer.")
+    removeWishlist(id: ID!): Boolean
+    addProductsToWishlist(input: AddProductsToWishlistInput): AddProductsToWishlistOutput
+    removeProductsFromWishlist(input: RemoveProductsFromWishlistInput): RemoveProductsFromWishlistOutput
+    updateProductsInWishlist(input: UpdateProductsInWishlistInput): UpdateProductsInWishlistOutput
+}
+
+type Wishlist {
+    id: ID
+    items: [WishlistItem]
+    items_count: Int
+    sharing_code: String
+    updated_at: String
+}
+
+input AddProductsToWishlistInput {
+    wishlist_id: ID!
+    wishlist_items: [WishlistItemInput!]!
+}
+
+input RemoveProductsFromWishlistInput {
+    wishlist_id: ID!
+    wishlist_items_ids: [ID!]!
+}
+
+input UpdateProductsInWishlistInput {
+    wishlist_id: ID!
+    wishlist_items: [WishlistItemUpdateInput!]!
+}
+
+input WishlistItemUpdateInput {
+    wishlist_item_id: ID
+    quantity: Float
+    selected_options: [String!]
+    entered_options: [EnteredOptionInput!]
+}
+
+type AddProductsToWishlistOutput {
+    wishlist: Wishlist!
+}
+
+type RemoveProductsFromWishlistOutput {
+    wishlist: Wishlist!
+}
+
+type UpdateProductsInWishlistOutput {
+    wishlist: Wishlist!
+}
+
+input WishlistItemInput {
+    sku: String
+    quantity: Float
+    parent_sku: String,
+    selected_options: [String!]
+    entered_options: [EnteredOptionInput!]
+}
+
+type WishlistOutput { #already implemented
+    items: [WishlistItem]
+    items_count: Int
+    name: String
+    sharing_code: String
+    updated_at: String
+}
+
+type WishlistItem { #already implemented
+    id: ID #BC change, was `Int`
```

Comment:
If it was released we need to check how strictly typed languages would treat this change. Most likely it will be incompatible.
Please investigate and let's discuss with Nishant how we should deliver it.

Reply:
We can introduce the field for example. It will cause an error anyway.
Another point is that we dont have UUID now, so what we can is to `base64_encode` actual entity `id` field.